examples.conf.template

```diff
@@ -0,0 +1,31 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Properties prefixed with accumulo.examples are not general Accumulo properties
+# and are only used by example code in this repository. All other properties are
+# general Accumulo properties parsed by Accumulo's ClientConfiguration
+
+instance.zookeeper.host=localhost:2181
+instance.name=your-instance-name
+accumulo.examples.principal=root
+accumulo.examples.password=secret
+
+# The following can be set when using Kerberos.   When set the password propety
```

Comment:
Could be `If you are using password authentication above, ignore the properties below as they only need to be set if you are using Kerberos authentication.`